docs/releases/5.0.7.txt

```diff
@@ -9,4 +9,5 @@ Django 5.0.7 fixes several bugs in 5.0.6.
 Bugfixes
 ========
 
-* ...
+* Fixed a bug in ``SplitArrayField`` where validation did not work properly 
+  when ``remove_trailing_nulls=True`` (:ticket:`35449`).
```

Comment:
Only release blocker bugs get back ported, so in this case, a release note is not needed 👍